test/python/test_resources_api.py

```diff
@@ -23,7 +23,9 @@ class TestResourcesApi(api_config.ConfiguredTest):
     """ResourcesApi unit test stubs"""
     def setUp(self):
         self.api = openapi_client.api.resources_api.ResourcesApi(self.client)
-        self.bad_auth_api = openapi_client.api.resources_api.ResourcesApi(self.bad_auth_client)
+        self.bad_auth_api = openapi_client.api.resources_api.ResourcesApi(
+            self.bad_auth_client
+        )
```

Comment:
Not breaking, just doesn't seem to fit in the PR. Was this a fix for linting errors?